servicekeeper-configsource/servicekeeper-configsource-common/src/main/java/io/esastack/servicekeeper/configsource/cache/ConfigCacheImp.java

```diff
@@ -38,7 +38,7 @@
 
     @Override
     public ExternalConfig configOf(ResourceId resourceId) {
-        final ExternalConfig config = configs.get(resourceId);
+        ExternalConfig config = configs.get(resourceId);
         if (config != null) {
             if (logger.isDebugEnabled()) {
                 logger.debug("Obtained {}'s config: {}", resourceId, config);
```

Comment:
Please add more logs to describe where the current config got from? directly by resourceId or fallback to arg value which can apply to all values?